examples/evolution_strategy.py

```diff
@@ -0,0 +1,109 @@
+"""Generates CMA-ES animation on sphere function.
+
+For gradient-based optimizers (inheriting from :class:`GradientOptBase`):
```

Comment:
If we're supporting gradient-based optimizers, then just call this file `optimizers.py`